app/models/submission.rb

```diff
@@ -91,6 +91,8 @@ def saveFile(upload)
       # Sanity!
       upload['file'].rewind
       File.open(path,"wb") { |f| f.write(upload['file'].read)}
+    elsif upload['local_submit_file']
+      File.open(path,"wb") { |f| f.write(IO.read(upload['local_submit_file']))}
```

Comment:
So upload['file'] is an actually file-like object, but upload['local_submit_file'] is a string?  Could we get a comment for that.